### ISSUE TITLE
scale.png does not reflect actual plugin range

### ISSUE THREAD

**Issue by falkTX** [AidaDSP/aidadsp-lv2#17]
the scale.png in use is a stub one, the values around it do match the real stuff it ought to be using

**Comment by spunktsch** [comment]
do you mean like pregain scale should be from -12 to 0 instead of the standard 0 -10?

**Comment by falkTX** [comment]
I mean this file https://github.com/AidaDSP/aidadsp-lv2/blob/main/rt-neural-generic/ttl/modgui/knobs/scale.png shows 0 min, 10 max.
this is incorrect. we need -8 to 8 for some knobs, and -12 to 12 in others.

**Comment by spunktsch** [comment]
yeah I also thought about that and came to the conclusion that its best to keep it simple 0 -10 and more traditional. 
That's what the scale is on most real amps and people know it. Most of the time the real values for that are in the manual. 

So front ui = simple, advanced ui = for in depth values without consulting the manual. 

I get it can be confusing when going in advanced view and the values are different but its always a problem of the mod platform that the scales are all over the place. Some are 0 -100 or 0 -1 or 0 - 100%. In the end it doesn't really matter because the scale is min to max. So to keep it consistent there should be one scale style. 

If anything I would just leave out the values in the front ui altogether and go with the tick marks.

**Comment by falkTX** [comment]
the values are as consistent as plugin developers choose to go with.
sometimes it is not possible to have a proper scale, so devs use 0-1.

for the gain related knobs, the scale is -8 to 8 which is well defined, so it could be in the graphics no?
without any labels the gui knobs lose their meaning.

an alternative is to display the knob value above or below the knob label/name.

**Comment by spunktsch** [comment]
Our approach is to go as simple as possible - the less numbers and details (on the front) the better.
It doesn't loose the meaning because its implied. It worked for 60+ years that knob position left is min/0, mid/50 and that right is max/100. 

> the values are as consistent as plugin developers choose to go with. sometimes it is not possible to have a proper scale, so devs use 0-1.
> 
that's the problem. It varies from dev to dev and is not consistent to the whole platform. Sometimes it makes sense to have it that way. But lets not discuss that here.

**Comment by falkTX** [comment]
I agree the less details the better. but I think still having the min and label is useful, rest can be skipped yes

**Comment by spunktsch** [comment]
then I would just skip the numbers altogether and go with ticks only. This is not a big issues but I think a detail that adds to the overall useability. Trust your ears not your eyes kind of a thing.

**Comment by falkTX** [comment]
sure, but I wonder on the upcoming desktop GUI needs it, since most hosts do not have a "generic" GUI view like MOD does. so depending on the DAW/Host, the user will never know what is the value they are changing, it is super weird to not have any value labels at all.

**Comment by spunktsch** [comment]
I would also go with the ticks on desktop and on grabbing the knob display the real value. Either via tooltip or changing the endpoints (switching images essentially).

**Comment by falkTX** [comment]
I like that idea! on MOD we can have this on the modgui as well without needing to plug custom js for the modgui.
we can have a hidden-by-default text label that follows the value of the knob, then those are shown when the user moves a knob (that is, "activates" the element, which is temporary)

```
<div class="mod-knob" title="">
    <!-- other stuff here -->
    <span class="mod-knob-current-value" mod-role="input-control-value" mod-port-symbol="DEPTH" >0 dB</span>
</div>
```

that should do it on the html side, then it is just a matter of styling

**Comment by falkTX** [comment]
scale related issue fixed, closing
